src/widgets/PageInfo.js

```diff
@@ -104,7 +110,7 @@ function renderInfoComponent(isEditingDisabled, columnIndex, rowData, rowIndex)
       break;
   }
   // If null or empty string, use single space to avoid squishing row
-  let infoString = rowData.info && String(rowData.info);
+  let infoString = (rowData.info || rowData.info === 0) && String(rowData.info);
```

Comment:
if rowData.info is falsy then it can't `=== 0`. If rowData is truthy then the second half is not evaluated. So the second half of this condition is redundant.